src/gobprepare/utils/graphql.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+from gobprepare.utils.requests import post_stream
+
+
+class GraphQL:
+
+    def __init__(self, host, endpoint, query):
+        self.host = host
+        self.endpoint = endpoint
+        self.query = query
+
+    @staticmethod
+    def item_to_list(item):
+        # Convert to list
+        items = item if isinstance(item, list) else [item]
+        yield from items
+
+    def __iter__(self):
+        items = post_stream(f'{self.host}{self.endpoint}', {'query': self.query})
+        for item in items:
+            yield from self.item_to_list(json.loads(item))
```

Comment:
renamed the method.

Reply:
That's right but it still is difficult to understand what happens. Why is it an array of arrays? Just comment an example of what is received of describe what happens should be enough I think.